main/src/library/String.flix

```diff
@@ -328,7 +328,7 @@ namespace String {
     /// Returns `None` if no character in `s` satisfies `f`.
     ///
     @Time(time(f) * length(s)) @Space(space(f))
-    pub def findIndexOfLeft(f: Char -> Bool, s: String): Option[Int32] =
+    pub def findIndexOfLeft(f: Char -> Bool & ef, s: String): Option[Int32] & ef =
```

Comment:
`String.forall` cannot be effect polymorphic, if `findIndexOfLeft` is not effect polymorphic too (or cast). Should `findIndexOfLeft` be effect polymorphic?

Reply:
Yes, lets make find effect polymorphic too.